setup.py

```diff
@@ -32,5 +32,8 @@
         'Topic :: Software Development :: Libraries :: Application Frameworks'
     ],
     zip_safe=False,  # don't use eggs
-    entry_points={"console_scripts": ["qdarkstyle=qdarkstyle.__main__:main"]}
+    entry_points={"console_scripts": ["qdarkstyle=qdarkstyle.__main__:main"]},
+    install_requires=[
+        'watchdog',
```

Comment:
This dependency may should be optional since common users not need that.

Reply:
This dependency may should be optional since common users not need that.

Reply:
What about add watchdog as optional dependencies[ (extra_requires](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies)) - as we will deliver compiled files for ordinary users (qdarkstyle as a dependency for another package), may those tools to generate the qss file should stay for developers? So, we keep only hard dependencies on install_requires in setup.py. (Thus, add qtsass to extra too) Agreed? @ColinDuquesnoy  @goanpeca

Reply:
For now, I would add watchdog and qtsass to the extra_requires and helpdev to the normal dependencies. There is no problem adding a python dependency that is used in the end-user python package. Anything else should go in the dev dependencies.

Reply:
I've added watchdog to extras_require :) Also to run examples.